org-gcal.el

```diff
@@ -591,7 +592,7 @@ TO.  Instead an empty string is returned."
                       end
                     (org-gcal--iso-previous-day end)))))) "\n"
 		    (when desc "\n")
-		    desc
+		    (replace-regexp-in-string "\*" "✱" desc)
```

Comment:
I can confirm that this works. I also changed the `*` to unicode bullet `•`, which looks nicer to my eyes.

@myuhe You willing to merge this thing if we update/respin this PR to address the merge conflicts and add the conditional?